tests/test_scripts/test_utils/test_s3client.py

```diff
@@ -0,0 +1,24 @@
+from time import sleep
+
+
+def test_client():
+    from scripts.utils.s3_client import Client
+
+    BGM = True
+    """bypass governance mode to remove files immediatly"""
+
+    client = Client()
+    assert client.prefix.startswith("sandbox")
+    client.put_json("test/test1.json", "test")
+    client.put_json("test/dir/test2.json", "test")
+    client.put_json("test/dir/test3.json", "test")
+    assert set(client.ls("test/")) == {"test1.json", "dir"}
+    client.mv_dir("test/", "test_b/", bypass_governance_mode=BGM)
+    sleep(3)
+    assert set(client.ls("test/")) == {}
```

Comment:
@jmetz this test fails here, any idea why removing objects from s3 does not work as I expect?

Reply:
Resolved by [4c1ed6a](https://github.com/bioimage-io/collection/pull/1/commits/4c1ed6a6bbcd5795d0105348fc2f769c077bad76)